packages/plugin/src/theia.d.ts

```diff
@@ -7172,6 +7172,47 @@ declare module '@theia/plugin' {
         resolveDebugConfiguration?(folder: WorkspaceFolder | undefined, debugConfiguration: DebugConfiguration, token?: CancellationToken): ProviderResult<DebugConfiguration>;
     }
 
+	/**
```

Comment:
something wrong with whitespace formatting, make sure to apply project formatting rules